src/System.IO/src/System/IO/MemoryStream.cs

```diff
@@ -474,41 +474,48 @@ public override int ReadByte()
             return _buffer[_position++];
         }
 
-
-        public override Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken)
+#if netcoreapp11
```

Comment:
`System.IO\src\System.IO.csproj` is referencing CoreLib ... it is why it compiles, and why I have said that netcoreapp1.1 is the "default".